### PR TITLE
Add JavaScript CLIs

### PR DESCRIPTION
**Pull request by Kikobeats**
Hello,

The [awesome-cli](https://github.com/Kikobeats/awesome-cli) group the best resources for creating your own CLI using JavaScript, and also a lot of goods CLI that combines the best of UNIX philosophy and user experience.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by Kikobeats** [comment]
sorry for that!

- [x] renamed `master` into `main`
- [x] `awesome-lint` is reporting 0 issues

I hope now is ready 🤞

**Comment by ozh** [comment]
There are still several points missing to comply to the PR requirements : 

* you didn't review (at least 2) open PR and state here what your reviews were (point 2 of the PR requirements)
* you didn't stick to the "short description about the project/theme of the list" guideline (point 5 of the PR reqs)
* you didn't add your list to the bottom of the appropriate category (point 6 of the PR reqs)

**Comment by Kikobeats** [comment]
Thanks for the feedback!

A short list of the current state

- [x] add a short description.
- [x] add your list to the bottom.
- [ ] review at least 2 PRs

**Comment by Kikobeats** [comment]
I totally forgo the item description – added!

**Comment by sindresorhus** [comment]
>  review at least 2 PRs

Bump

**Comment by Kikobeats** [comment]
Hey, thanks for taking a look over this!

- [x] review at least 2 PRs

https://github.com/sindresorhus/awesome/pull/2020#issuecomment-854770260
https://github.com/sindresorhus/awesome/pull/2025#issuecomment-854774279

File: readme.md [hunk 1]
```diff
@@ -148,6 +148,7 @@
 	- [npm scripts](https://github.com/RyanZim/awesome-npm-scripts#readme) - Task runner.
 	- [30 Seconds of Code](https://github.com/30-seconds/30-seconds-of-code#readme) - Code snippets you can understand in 30 seconds.
 	- [Ponyfills](https://github.com/Richienb/awesome-ponyfills#readme) - Like polyfills but without overriding native APIs.
+	- [CLI](https://github.com/Kikobeats/awesome-cli#readme) - Create your own CLI using JavaScript or be inspired by bests of UNIX philosophy and UX.
```
Comment (sindresorhus) [inline readme.md:151]: The description should describe the topic, not what the list contains.

**Comment by sindresorhus** [comment]
The list repo name and title does not match the PR title or its contents.

**Comment by sindresorhus** [comment]
I must also question the curation. For example, this package has not been maintained for 5 years: https://github.com/freeall/single-line-log

**Comment by sindresorhus** [comment]
The initial description in your list needs to be more verbose.

**Pull request closed without merging**